bucket/azure-kubelogin.json

```diff
@@ -0,0 +1,26 @@
+{
+    "version": "0.0.11",
+    "description": "A client-go credential (exec) plugin implementing azure authentication.",
+    "homepage": "https://github.com/Azure/kubelogin/",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/Azure/kubelogin/releases/download/v0.0.11/kubelogin-win-amd64.zip",
+            "hash": "8d509b64638bf2acba247b5333b8dc1f16a578ecb1917b562ec1ee938e49d85f"
+        }
+    },
+    "bin": "bin\\windows_amd64\\kubelogin.exe",
+    "checkver": {
+        "github": "https://github.com/Azure/kubelogin"
+    },
```

Comment:
```suggestion
    "checkver": "github",
```

Reply:
I've changed this, but I'm not convinced that the suggested pattern is better than the explicit one provided.